### ISSUE TITLE
Not correct version of `2.2.1` in https://pub.dev/packages/dio/versions cause error.

### ISSUE THREAD

**Issue by jerryzhoujw** [flutterchina/dio#464]
### New Issue Checklist

- [ ] I have searched for a similar issue in the [project](https://github.com/flutterchina/dio/issues) and found none

### Issue Info

| Info                           | Value                                                 |      |
| ------------------------------ | ----------------------------------------------------- | ---- |
| Platform Name                  | e.g. flutter / ios / android                          |      |
| Platform Version               | e.g.  1.5.0 / 12.0 / 9.0                              |      |
| Dio Version                    | e.g.  2.1.0 / 1.0.17                                  |      |
| Android Studio / Xcode Version | e.g. Android Studio 3.3.2 / Xcode 10.2.1              |      |
| Repro rate                     | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo prj        | e.g. does it happen with our demo project?            |      |
| Demo project link              | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

1. These config will cause get the `dio 2.2.1`， but `UploadFileInfo` already removed in that version.
> pubspec.yaml 

```dart
  dio: ^2.1.16
```

2. I think this issue can be resolved by remove the `dio 2.2.1` in `https://pub.dev/packages/dio/versions`, but need someone who have permission to make change on that page.

3. I can solve this issue in my code by change from  `dio: ^2.1.16` to `  dio: 2.1.16`.

**Comment by wendux** [comment]
I just released a 2.2.2 version, which is compatible with 2.1. This is a quick fix.
